src/app/jobs/inactivevoice.job.ts

```diff
@@ -2,14 +2,14 @@ import { IContainer } from '../../common/types';
 import { Job } from '../../common/job';
 
 export class InactiveVoiceJob extends Job {
-  public interval: number = 1000 * 60 * 10; // Every 10 mintues
-  public name: string = 'Inactive Voice';
+  public override interval: number = 1000 * 60 * 10; // Every 10 mintues
+  public override name: string = 'Inactive Voice';
 
   constructor() {
     super();
   }
 
-  public async execute(container: IContainer) {
+  public override async execute(container: IContainer) {
```

Comment:
`interval`, `name`, and `execute` should all be required in a job

Reply:
> `interval`, `name`, and `execute` should all be required in a job

They aren't, `job.ts` provides default values for all of those fields:
```typescript
export abstract class Job implements IJob {
  public name: string = '';
  public interval: number = 60000;
  public execute(container?: IContainer) {}
}
```